readme.md

```diff
@@ -53,7 +53,71 @@ Directory names should be lowercase. Filenames should be singular.
 *todo: evaluate common prefix like `pagelib` and BEM naming.*
 
 ## Development setup and workflow
-*Coming soon.*
+The Android and iOS Wikipedia apps make use of the `wikimedia-page-library`. At some point their respective build phases invoke `npm install` which causes a *published* version of this library to be retrieved and used in their respective builds. 
```

Comment:
I see your point about side-by-side. 

I guess for me the majority of the time I'm only looking at one file at a time, and if not I'm usually using a diff viewer. 

I do have a big screen. 

But even on a smaller screen I find hardcoded line breaks problematic because if you do reduce your window width and your editor is configured to wrap lines, the line breaks wreak havoc when the editor tries to wrap to the window width if the editor window width is reduces to be narrower than the (somewhat arbitrary) character width restriction... if that makes sense.